src/transaction/log_impl.h

```diff
@@ -737,7 +737,12 @@ struct log_global
   void initialize_log_prior_sender ();
   void finalize_log_prior_sender ();
   // *INDENT-OFF*
-  cublog::prior_sender &get_log_prior_sender ();
+  cublog::prior_sender &
+  get_log_prior_sender()
+  {
+    assert (m_prior_sender != nullptr);
+    return *m_prior_sender;
+  }
```

Comment:
Now `LOG_RESET_APPEND_LSA` use this function that a test fails to build with the msg:
```
CMakeFiles/test_prior_list_serialize.dir/__/__/src/transaction/log_append.cpp.o: In function `LOG_RESET_APPEND_LSA(log_lsa const*)':
/home/cubrid/cubrid/lets/src/transaction/log_append.cpp:188: undefined reference to `log_global::get_log_prior_sender()'
```

Reply:
I could put the implementation, `log_global.c`, but causes more errors since it has more dependencies on other source files.

Reply:
It is ok like this.